package.json

```diff
@@ -4,8 +4,8 @@
     "postinstall": "node script/get-quasar.js",
     "clean": "rimraf output && rimraf .pulp-cache && rimraf test/tmp",
     "build": "pulp build -- --censor-lib --strict",
-    "property-tests": "pulp test --test-path test/src --main Test.Property.Main",
-    "test": "pulp test --test-path test/src --main Test.Property.Main && pulp test --test-path test/src"
+    "test-fast": "pulp test --test-path test/src --main Test.Fast",
```

Comment:
The other option would be to rename this one to `test`, and rename the current `test` to `test-full` / `test-feature` or whatever.